README.md

```diff
@@ -154,4 +154,21 @@ We use SSH for remote login to server. Linux knowledge is highly recommended. Re
 For Windows: Use PuTTY
 For Mac / Linux: Use OpenSSH
 
+#### Windows Installation
+
+The suggested client is PuTTY so this will only cover PuTTY setups.
+
+You cannot log in to SSH via putty using password(doesnt support publickey), OpenSSH doesn't seem to have this issue
+Detailed Procedure:
+1. Make sure you have PuTTY and puttygen installed (puttygen should be bundled in PuTTY installation)
+2. Request for a key from your department leader or one of the founders (yes you actually need to verify yourself before you can login)
```

Comment:
No we don't create keys for you. You generate your own keys and give the public part of the pair to us.